.circleci/config.yml

```diff
@@ -84,7 +91,9 @@ jobs:
             yarn postdeploy
       - run:
           name: Start Testing Server
-          command: yarn start || true
+          command: |
+            sudo apt-get update && sudo apt-get install -y libvips-dev --no-install-recommends
```

Comment:
-2 This was wrong in retrospect: sharp installs libvips in the node_modules folder itself, so it is only required on the system if something is wrong with the library resolution.

Reply:
the goal here was to build a docker container without node_modules, so we had to install the binaries separately vs. copying them from node_modules and updating the symlink. was this not working?

Reply:
We did not update the dokku container for staging/production and the loader didn't pick up the .so file in the node_modules folder anymore. If that was the goal, then these containers need to be updated first.

Reply:
that confirms it, i'm an idiot.
i'll need to think more about how to do this. containerizing & transferring the image is taking a great deal of time because node_modules accounts for ~1.5GB IIRC, whereas bundling everything & having a couple external native `.node` files comes in at around 150MB total.

I think the best way forward is to probably just write a script that copies over sharp from node_modules. Then, we read the postinstall script to find out what it's doing to the `$PATH` and adjust links accordingly.